### ISSUE TITLE
Many demos have missing page title (except - MS Edge Dev postfix)

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#335]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/typedarrays/

This is WCAG 2.4.2 https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html

**Comment by dstorey** [comment]
Also https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/familysearch/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/userselect/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/setimmediatesorting/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/analyzeperformance/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/chess/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/picture/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/coloringbook/

**Comment by dstorey** [comment]
This one could be better as it is all one word and probably reads badly https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/twilioortc/
Same with this https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/ortcdemo/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/math/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/@supports/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/eme/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/mandelbrot/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/webaudiotuner/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/microphone/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/css3filters/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/blobbuilder/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/photocapture/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/compatinspector/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/audiomixer/
